data/Tables/C3/CSp6.1.jl

```diff
@@ -11390,24 +11390,12 @@ charparams = [
 classparamindex = var_index.([i1, i2, i3, i4])
 charparamindex = var_index.([k1, k2, k3, k4])
 
-information = raw"""- Information about the generic character table of $\mathrm{CSp}_6(q)$, $q$ odd
-
-- CHEVIE-name of the table: `CSp6.1`
+information = raw"""The generic character table of $\mathrm{CSp}_6(q)$, $q$ odd
 
 - The computation of this table is explained in [Lbe93](@cite).
 
-- This table is very big. The best way to work with it
-  is to produce one time a file in ``Maple-internal-format``
-  (in a new cheviem-session say:
-> GenCharTab(``CSp6.1``);save ``CSp6_1.m``;quit;
-  and ask your system-administrator to copy  the file
-  CSp6_1.m into the CHEVIE-directory tables/C3/CSp6_1.m).
-  Now you can read in this table with the CHEVIE-command
-> GenCharTab(``CSp6_1.m``);
-  This is much faster and uses much less memory.
-
 - The first twelve character types contain the unipotent characters.
-  Specialize  the parameter kK1 to a multiple of q-1 e.g., kK1=0.
+  Specialize  the parameter `k1` to a multiple of $q-1$ e.g., `k1=0`.
```

Comment:
I wonder why one can't replace all occurrences of `k1` by `k1*(q-1)` and then not restrict its values?